readthedocs/projects/models.py

```diff
@@ -825,7 +828,10 @@ def vcs_repo(self, version=LATEST, environment=None):
         if not backend:
             repo = None
         else:
-            repo = backend(self, version, environment)
+            repo = backend(
+                self, version, environment,
```

Comment:
we could use named args here, just to not depend on the order

Reply:
sure!